### ISSUE TITLE
RME Showcase

### ISSUE THREAD

**Issue by BastienDuplessier** [RMEx/RME#403]
Il serait peut être intéressant d'avoir un "RME Showcase".
Ce serait une page qui rassemblerait des systèmes réalisés à l'aide de RME avec potentiellement des liens vers les jeux / guides.
Qu'est ce que vous en dites ?

**Comment by xvw** [comment]
Yep. Je pense que ça s'inscrit dans "une refonte complète" du site web :)

**Comment by BastienDuplessier** [comment]
Ce sera à discuter alors :)
